paraflow-metaserver/src/main/java/cn/edu/ruc/iir/paraflow/metaserver/utils/MetaConfig.java

```diff
@@ -46,7 +46,7 @@ public boolean validate()
             boolean driver = mDriver.find();
             //dbHost format check
             String dbHost = getDBHost();
-            String regExHost = "^[a-zA-Z]+:[a-zA-Z]+://[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{1,5}(/[a-zA-Z]+)+$";
+            String regExHost = "^[a-zA-Z]+:[a-zA-Z]+://(([0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3})|[a-zA-Z]+):[0-9]]{0,5}(/[a-zA-Z]+)+$";
```

Comment:
extra `]` in regex. Cannot match all cases, for example: jdbc:postgresql://presto01:9000/metabase